### DISCUSSION TITLE
repoVersion always updated + added in Increment Version Number workflow

### DISCUSSION THREAD

**Discussion by dsaveyn** [microsoft/AL-Go#1590] [Ideas]
We have a multi project repository which has the following state:

repoVersion = 23.0

- App 1: 
  - app.json ==> 23.0
- App 2: 
  - app.json ==> 23.0
- App 3: 
  - app.json ==> 23.0

The repoVersion property is not configured on project level, only on repo level.

We want to update the repoVersion only when there's a new major release.

Suppose we need to release a new feature in App 3 which relies on a change in App 1, we would like to update only the version numbers of those apps affected by the change. 

So for example, our desired result would be:

- App 1:
  - app.json ==> 23.1
- App 2:
  - app.json ==> 23.0 (no changes done)
- App 2:
  - app.json ==> 23.1

But, when we use the Increment Version Number workflow, to achieve the above scenario, the following happens:

repoVersion = 23.0

- App 1:
  - **repoVersion (new): 23.1**
  - app.json ==> **23.1**
	
- App 3:
  - **repoVersion (new): 23.1**
  - app.json ==> **23.1**

So the repoVersion is added to the affected projects, something we don't want since it was already configured on repo level. 

Also, any chance we could get a parameter to define if we want to update the repoVersion or not?

**Comment by freddydk** [comment]
The RepoVersion is actually a project version and it covers all apps in a specific project.
But, we will discuss whether we can make this better.
